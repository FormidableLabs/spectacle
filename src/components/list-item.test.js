import ListItem from './list-item';
import React from 'react';
import { mount } from 'enzyme';

describe('<ListItem />', () => {
  test('should render correctly', () => {
    const context = {
      styles: {
        components: {
          listItem: {
            color: '#ff0'
          }
        }
      }
    };
    const wrapper = mount(<ListItem>Dog</ListItem>, { context });
    expect(wrapper).toMatchSnapshot();
  });
});
