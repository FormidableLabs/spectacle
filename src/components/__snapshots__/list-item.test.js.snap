// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> should render correctly 1`] = `
<ListItem>
  <Styled(li)
    className={undefined}
    styles={
      Array [
        Object {
          "color": "#ff0",
        },
        Object {},
        Object {},
        undefined,
      ]
    }
  >
    <li
      className="css-tr63ps"
    >
      Dog
    </li>
  </Styled(li)>
</ListItem>
`;
